docs/source/conf.py

```diff
@@ -31,15 +31,16 @@
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '4.2.0'
+needs_sphinx = '4.3.2'
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.napoleon',
-    'sphinx.ext.intersphinx'
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.linkcode'
```

Comment:
Ah, furo doesn't have an "edit on GH" button … maybe you can suggest that as well? would like that better than the `[source]` buttons that at least currently all lead to the same location. and sphinx-doc/sphinx#1556 doesn't look like it's gonna be closed soon xD

Reply:
well there is a show source button at the bottom, but that only leads to the local `.rst.txt` file

Reply:
jup, saw that. anyway, I'm okay with the way it is :)

Reply:
https://github.com/pradyunsg/furo/pull/346

for reference